addons/website_event/controllers/main.py

```diff
@@ -185,49 +164,41 @@ def event_register(self, event, **post):
 
     @http.route('/event/add_event', type='http', auth="user", methods=['POST'], website=True)
     def add_event(self, event_name="New Event", **kwargs):
-        return self._add_event(event_name, request.context, **kwargs)
+        event = self._add_event(event_name)
+        return request.redirect("/event/%s/register?enable_editor=1" % slug(event))
 
-    def _add_event(self, event_name=None, context={}, **kwargs):
+    def _add_event(self, event_name=None):
         if not event_name:
             event_name = _("New Event")
-        Event = request.registry.get('event.event')
         date_begin = datetime.today() + timedelta(days=(14))
         vals = {
             'name': event_name,
-            'date_begin': date_begin.strftime('%Y-%m-%d'),
-            'date_end': (date_begin + timedelta(days=(1))).strftime('%Y-%m-%d'),
+            'date_begin': fields.Date.to_string(date_begin),
+            'date_end': fields.Date.to_string((date_begin + timedelta(days=(1)))),
             'seats_available': 1000,
         }
-        event_id = Event.create(request.cr, request.uid, vals, context=context)
-        event = Event.browse(request.cr, request.uid, event_id, context=context)
-        return request.redirect("/event/%s/register?enable_editor=1" % slug(event))
+        return request.env['event.event'].create(vals)
 
     def get_formated_date(self, event):
-        context = request.context
-        start_date = datetime.strptime(event.date_begin, tools.DEFAULT_SERVER_DATETIME_FORMAT).date()
-        end_date = datetime.strptime(event.date_end, tools.DEFAULT_SERVER_DATETIME_FORMAT).date()
-        month = babel.dates.get_month_names('abbreviated', locale=context.get('lang', 'en_US'))[start_date.month]
-        return _('%(month)s %(start_day)s%(end_day)s') % {
-            'month': month,
-            'start_day': start_date.strftime("%e"),
-            'end_day': (end_date != start_date and ("-"+end_date.strftime("%e")) or "")
-        }
+        start_date = fields.Datetime.from_string(event.date_begin).date()
+        end_date = fields.Datetime.from_string(event.date_end).date()
+        return ('%s %s%s') % (start_date.strftime("%b"), start_date.strftime("%e"), (end_date != start_date and ("-"+end_date.strftime("%e")) or ""))
```

Comment:
Is it OK if it always give abbreviated names in English(My server language is en_us) even though my context language is fr_FR, why we removed babel, is there any reason ?